browser/index.html

```diff
@@ -214,6 +214,9 @@
                                                     <li v-if="'qemu-secex' in build.meta.images">
                                                         QEMU Secure Execution: <span v-html="getImageUrl(build, 'qemu-secex')"></span>
                                                     </li>
+                                                    <li v-if="'ignition-gpg-key' in build.meta.images">
+                                                        Ignition Protection GPG Key: <span v-html="getImageUrl(build, 'ignition-gpg-key')"></span>
```

Comment:
How about:

> QEMU Secure Execution Ignition PubKey

?

Reply:
Sounds fine to me too. To be honest I was struggling with what to call it.